### PR TITLE
[JOLINT-27] inclusion score component

### PR DESCRIPTION
**Pull request by xxrobone**

### CODE REVIEW

File: app/dashboard/_components/cards/InclusionScore/InclusionScore.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import React from 'react'
```
Comment (ell-ska) [inline app/dashboard/_components/cards/InclusionScore/InclusionScore.tsx:1]: no need to import react

File: app/dashboard/_components/cards/InclusionScore/InclusionScore.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import * as Progress from '@radix-ui/react-progress'
```
Comment (ell-ska) [inline app/dashboard/_components/cards/InclusionScore/InclusionScore.tsx:2]: empty line between local imports and libraries

File: app/dashboard/_components/cards/InclusionScore/InclusionScore.tsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+            className={cn(
+              'absolute h-full w-full rounded-sm',
+              score >= benchmark ? 'bg-green' : 'bg-red',
+            )}
+            style={{ transform: `translateX(-${100 - score}%)` }}
+          />
+        </Progress.Root>
+        <div>
+          {trendData.map(({ trend, text }) => (
+            <div key={trend} className='flex gap-2  first-of-type:pb-4'>
```
Comment (ell-ska) [inline app/dashboard/_components/cards/InclusionScore/InclusionScore.tsx:49]: no double space

**Review by ell-ska — APPROVED** [review]
Nice work! just some minor syntax things

**Comment by idaakermark** [comment]
Tested on dev: The "bar" for the inclusion score has a border radius of "rounded-sm" it should have a border of "rounded-md" other than that it looks good.

**Comment by xxrobone** [comment]
@idaakermark I have fixed the border-radius now, please check testing again. :D

**Pull request merged**